frontend/app/src/components/EvmNativeTokenBreakdown.vue

```diff
@@ -35,38 +39,47 @@ const breakdowns = computed(() => {
 
 const { currencySymbol } = storeToRefs(useGeneralSettingsStore());
 
-const tableHeaders = computed<DataTableHeader[]>(() => {
-  const headers: DataTableHeader[] = [
+const sort: Ref<DataTableSortColumn | DataTableSortColumn[] | undefined> = ref({
+  column: 'balance.usdValue',
```

Comment:
since there are lots of other places where `Balance { amount, usdValue }` is used, should I do complete refactor making it a flat object, or optionally add the same values (?amount, ?usdValue) to the root type.

from:
```typescript
interface XpubAccountWithBalance extends XpubAccount, HasBalance {};
```
to

```typescript
interface XpubAccountWithBalance extends XpubAccount {
  amount: BigNumber;
  usdValue: BigNumber;
};
```
or to

```typescript
interface XpubAccountWithBalance extends XpubAccount, HasBalance {
  amount?: BigNumber;
  usdValue?: BigNumber;
};
```

for the first option, we'll have to flatten all use cases, and this might also break the tables where they are used.

for the second, will be a gradual process, since the flattened keys are not required, until we are fully done with migrating other parts where we use this interface, then we can remove `HasBalance`, and just extend `Balance`.

Reply:
I think the easiest way to migrate is start with edges and then remove the mapping. What do you think?